gulpfile.js

```diff
@@ -130,6 +130,10 @@ gulp.task('img', function() {
   return gulp.src(config.src.img).pipe(gulp.dest(config.dest.img));
 });
 
+gulp.task('templatedata', function() {
+  return gulp.src(config.src.templateData).pipe(gulp.dest(config.dest.templateData));
```

Comment:
instead of calling them data - i guess you could give it a different foldername like rest/ or api/ etc, but the idea sounds good to me.